.travis.yml

```diff
@@ -19,6 +19,7 @@ php:
   - 5.5
   - 5.6
   - 7.0
+  - 7.1
```

Comment:
Testing under PHP 7.1 is already done: see these lines in the `matrix` section:
```
    - php: 7.1
      env:
        - CODECOVERAGE="yes"
        - UPDATEDOCS="yes"
```
where, in addition to testing PHP 7.1, we also set a couple of environment variables.